.github/workflows/tests.yml

```diff
@@ -0,0 +1,158 @@
+# This is a workflow to run our unit and integration tests for windows and mac
+
+name: dbt Tests
+
+# Triggers
+on:
+  # Triggers the workflow on push or pull request events but only for the develop branch
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: 
+      - 'develop' 
+      - '*.latest' 
+      - 'pr/*'
+      - 'releases/*'
+  pull_request_target:  
+    branches: [ develop ]
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  UnitTest:
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Python
+      uses: actions/setup-python@v2.2.2
+      with:
+        python-version: '3.7'
```

Comment:
We probably want to set the python version in the matrix for 3.6, 3.7, and 3.8

Reply:
+3.9, which is supported by `dbt-core==0.19.0`, and by all plugins in this repo as of v0.20

Reply:
I think I'll save this for the next iteration on this file since I had 3.9 and it was failing so I need to dig into that. To keep this simple for the moment, I'll do just version 3.7 and then build upon it before actually depending on this